src/sockets/matchmaking.ts

```diff
@@ -0,0 +1,33 @@
+export class Matchmaking {
+    userQueue: string[];
+
+    constructor() {
+        this.userQueue = [];
+    }
+
+    // Add user to queue
+    addUserToQueue(username: string): void {
+        if (!(username in this.userQueue)) {
```

Comment:
Could you use Array.includes() pls? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes